s3prl_vc/bin/train.py

```diff
@@ -376,17 +383,19 @@ def _plot_and_save(
         with torch.no_grad():
             all_hs, all_hlens = self.upstream_model(xs, ilens)
             hs, hlens = self.upstream_featurizer(all_hs, all_hlens)
-            outs, _ = self.model(hs, hlens, spk_embs=spembs, f0s=f0s)
+            outs, ynorm, _ = self.model(hs, hlens, spk_embs=spembs, f0s=f0s)
 
         for idx, (out, olen, y) in enumerate(zip(outs, olens, ys)):
             out = out[:olen]
             y = y[:olen]
-            _plot_and_save(
-                out.cpu().numpy(),
-                dirname + f"/outs/{idx}_out.png",
-                ref=y.cpu().numpy(),
-                origin="lower",
-            )
+
+            if self.config["model_type"] == "Taco2_AR":
```

Comment:
Same as my question in the last round of review -- is there a specific reason why the diffusion model cannot generate dev set samples?

Reply:
The if condition is only for generating the plots of the attention weights, so the code should generate samples for both taco and diffusion models here.

Reply:
Wait I thought "out" is the generated mel? So both Taco2_AR and Diffusion should generate these?